go-controller/pkg/ovn/egressip.go

```diff
@@ -2307,3 +2387,14 @@ func getPodNamespaceAndNameFromKey(podKey string) (string, string) {
 	parts := strings.Split(podKey, "_")
 	return parts[0], parts[1]
 }
+
+// getFirstNonNilNamespace must return a non-nil pointer to a Namespace otherwise panic
+func getFirstNonNilNamespace(n1, n2 *v1.Namespace) *v1.Namespace {
+	if n1 != nil {
+		return n1
+	}
+	if n2 != nil {
+		return n2
+	}
+	panic("getNonNilNamespace(): received two nil Namespace pointers")
```

Comment:
IMHO it would be better if we could avoid the panic, and properly handle the error.